src/instructlab/sdg/generate_data.py

```diff
@@ -236,7 +254,8 @@ def generate_data(
     use the SDG library constructs directly, and this function will likely be removed.
 
     Args:
-        pipeline: This argument may be either an alias defined by the sdg library ("simple", "full"),
+        pipeline: This argument may be either an alias defined in a user or site "data directory"
```

Comment:
What's the easiest way for someone to know what the "user or site data directory" is? I'm thinking ahead to what we'd put in the CLI help text.

Reply:
It's the usual `$XDG_DATA_HOME:$XDG_DATA_DIRS` semantics?